content/influxdb/clustered/install/set-up-cluster/configure-cluster/_index.md

```diff
@@ -6,22 +6,20 @@ description: >
 menu:
   influxdb_clustered:
     name: Configure your cluster
-    parent: Install InfluxDB Clustered
-weight: 130
-related:
-  - /influxdb/clustered/admin/upgrade/
+    parent: Set up your cluster
+weight: 220
+aliases:
+  - /influxdb/clustered/install/configure-cluster/
 ---
 
 InfluxDB Clustered deployments are managed using Kubernetes and configured using
 a YAML configuration file. 
 Apply configuration settings to your cluster by editing and applying a
 Kubernetes custom resource (CRD) called `AppInstance`.
-The AppInstance CRD is defined in a YAML file (use example-customer.yml as a
-template) or, if using the InfluxDB Clustered Helm chart, is provided by the
-chart and configured in a `values.yaml` file.
+The AppInstance CRD is defined in a YAML file (use the `example-customer.yml`
```

Comment:
My preference here would be to downplay the Helm chart and emphasize the operator. For customers who don't have a hard "we must use Helm" requirement, it would be best if they use the operator since that avoids having to deal with an unnecessary layer of indirection when we end up on support calls.

Reply:
Sound good. I'll update this content.